src/tooltip/README.md

```diff
@@ -175,7 +181,7 @@ A Tooltip component for displaying informative popover information.
 |------|------|-------------|
 | `activateOn` | `TooltipActivationT \| TooltipActivationT[]` | Activate the tooltip through one or more interactions. Defaults to `['hover', 'focus']`. |
 | `align` | `TooltipAlignT` | How to align the tooltip. Defaults to `top`. |
-| `children` | `React.ReactChild` | The children that the tooltip belongs to. Must be a single React.child. |
+| `children` | `ReactElement<any, string \| React.JSXElementConstructor<any>> \| undefined` | The children that the tooltip belongs to. Must be a single React.child. |
```

Comment:
The documentations should be change. Probably to something like:

The children that the tooltip belongs to. Must be a single React element